test/fixtures/sample-file.ts

```diff
@@ -0,0 +1,3 @@
+// This is just a sample file, because ESLint requires a file to run
+
+export default () => 'Hello World';
```

Comment:
If we want to test the eslint rules, I would suggest to set up the sample file importing its `CLIEngine`, pass the config file to the `CLIEngine` config and run `executeOnText`.

Maybe something like this

```ts
import 'fs';
import 'eslint';

function getErrors(configFile, fileToTest) {
    const CLIEngine = eslint.CLIEngine;

    const cli = new CLIEngine({
        configFile: configFile,
    });

    return cli.executeOnText(fs.readFileSync(fileToTest, 'utf8'));
}
```

Then, in jest we could do something like this

```ts
describe('Validate ESLint configs', () => {
    ['myfirstconfig.js', 'mysecondconfig.js'].forEach((file) => {
        it(`load config ${file} in ESLint to validate all rules are correct`, () => {
            expect(getErrors(file, 'index.js').results[0].messages).toEqual([]);
        });
    });
});
```